python/lsst/pex/config/config.py

```diff
@@ -1055,6 +1063,23 @@ def __new__(cls, *args, **kw):
         instance.update(__at=at, **kw)
         return instance
 
+    def copy(self) -> Config:
+        """Return a copy of this config.
```

Comment:
Can you document whether this is a shallow or deep copy? I assume it's not possible to affect the original in any way? (some copies seem to be relatively shallow)